src/main/java/net/fabricmc/loom/providers/MinecraftAssetsProvider.java

```diff
@@ -76,18 +80,18 @@ public static void provide(MinecraftProvider minecraftProvider, Project project)
 			}
 		}
 
-		ProgressLogger progressLogger = ProgressLogger.getProgressFactory(project, MinecraftAssetsProvider.class.getName());
-		progressLogger.start("Downloading assets...", "assets");
+		project.getLogger().lifecycle(":downloading assets...");
+
+		Stack<ProgressLogger> loggers = new Stack<>();
+		ExecutorService executor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() / 2);
```

Comment:
Pretty sure `Runtime.getRuntime().availableProcessors() / 2` will give you 0 when on a single core processor.

Reply:
Sloved it.